### DISCUSSION TITLE
Allow open/close of a group by clicking on whole header.

### DISCUSSION THREAD

**Discussion by cschanck** [Able-Bear-Software/tab-nodes-tree#60] [Ideas / Feature Request]
The '>' is a small target, I would really like it if I could bang the header and collapse/open the entire group.

**Comment by zeroarst** [comment]
Hey @cschanck 
I like this idea. I guess a new setting could be added for the user to choose whether they want to do that by single click (tab needs to be active first), or double click.

**Reply by zeroarst** [reply]
Ticket created: #62

**Comment by Tab-Nodes-Tree** [comment]
Released in [1.5.0](https://github.com/Able-Bear-Software/tab-nodes-tree/discussions/84).
